static/less/style.less

```diff
@@ -194,20 +194,66 @@ img {
         margin-left: 1.0rem;
         margin-right: 1.0rem;
       }
+      @media only screen and (min-device-width : 320px) and (max-device-width : 768px) {
+        float: none !important;
```

Comment:
I personally don't like menu items in dropdown to be `center`ed, and instead be `left` aligned for LTR, and `right` aligned for RTL.

Reply:
Looks good centered to me. hehehe

Reply:
As I said, I'm fine either way. We already have `custom_css` ability and SCSS you mentioned.